syncwatcher.go

```diff
@@ -459,6 +450,23 @@ func relativePath(path string, folderPath string) string {
 	return path
 }
 
+// Returns a function to test whether a file should be ignored
+// Ignorefile must contain the absolute path to ".stignore". The returned
+// function takes the path of the file to be tested relative to its folders root
+func createIgnoreTest(folderPath string) func(relPath string) bool {
+	ignores := ignore.New(false)
+	ignores.Load(filepath.Join(folderPath, ".stignore"))
+	return func(relPath string) bool {
+		if strings.HasPrefix(relPath, versionsFolder) { return true }
```

Comment:
I want to match .stversions/* though. So what I need is to add a path separator after  `versionsFolder`. I did not find any elegant way, all filepath method strip the trailing file separator. A solution would be  (ugly):  
`strings.HasPrefix(relPath, filepath.FromSlash(versionsFolder + '/'))`

You certainly don't need to apologize, I would have been screwed without your help with the amount of mistakes I produced without even noticing...